cli/src/main/kotlin/commands/AnalyzerCommand.kt

```diff
@@ -286,3 +296,10 @@ class AnalyzerCommand : CliktCommand(name = "analyze", help = "Determine depende
         severityStats.print().conclude(config.severeIssueThreshold, 2)
     }
 }
+
+private fun AnalyzerConfiguration.computeEnabledPackageManagers(): Set<PackageManagerFactory> {
```

Comment:
Nit: "compute" is maybe a bit pretentious. Maybe just one of these: "get", "determine".

Reply:
Used `determine` because `get` would override the generated getter for the property with the same name.